### DISCUSSION TITLE
Play icon lingers too long in fullscreen when moving mouse?

### DISCUSSION THREAD

**Discussion by PaperOrb** [FreeTubeApp/FreeTube#6241] [Q&A]
When in fullscreen, when you move your mouse, the play icon lingers for a really long time in freetube v 0.22 compared to older freetube versions. Is there a way to change this behavior?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No way to change this behavior at the time but you could disable the button if you want in the settings, Settings > Player > Display Play Button In Video Player
